server/db.js

```diff
@@ -81,6 +82,7 @@ function initDatabase(fp, puzzles) {
 const addColumns = (db) => {
   const columnsToAdd = {
     User: [
+      { name: "is_oauth", typeDef: "INTEGER"},
```

Comment:
Probably rename to something like `oauth_id` or the like. Then store the Google ID in this field

Reply:
Done x3.